src/test/java/uk/co/evoco/webdriver/configuration/WebDriverConfigTests.java

```diff
@@ -91,4 +91,20 @@ public void testSettingBaseUrlWithBadUrlGivesGoodException() {
             webDriverConfig.setBaseUrl("bad-url");
         });
     }
+
+    @Test
+    public void testExceptionsWaitTimeOutReturnValue() throws JsonProcessingException {
+        String configJson = "{\"timeout\": \"30\",\"exceptionsWaitTimeOut\": \"5\"}";
+        WebDriverConfig webDriverConfig = JsonUtils.fromString(configJson, WebDriverConfig.class);
+        assertThat(webDriverConfig.getExceptionsWaitTimeOut(), is(5));
+
+    }
+
+    @Test
+    public void testExceptionsWaitTimeOutNotReturnValue() throws JsonProcessingException {
+        String configJson = "{\"timeout\": \"30\"}";
```

Comment:
I don't really like strings like this in tests, I'd rather that this was a fixture file in test resources folder or a serialised object

Reply:
created fixture to test this